LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Refract Project
+Copyright (c) 2015 Mark W. Foster
```

Comment:
> I believe need to be legal entities

Yes, was mostly curious of legalities here. Probably should change https://github.com/refractproject/refract-spec license then.